kinesis/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/sinks/KinesisSink.scala

```diff
@@ -410,6 +451,8 @@ object KinesisSink {
             executorService,
             sqsClientAndName
           )
+        ks.checkKinesisHealth()
+        ks.checkSqsHealth()
```

Comment:
Can you confirm these threads don't interfere with shutting down the app cleanly?

The `KinesisSink` has a `shudown` method, and instinct tells me we should use it to stop the running threads.  Or, run the threads on a executorService, and use the shutdown hook to close the exeuctorService.

But...... whatever works is OK for me.

Reply:
Good point, I hadn't thought of that, because these checks happen at startup and should be long done when the collector exits, but we never know, let me update !